### PR TITLE
Advances in UI Data Sources

### PR DESCRIPTION
**Pull request by soucolline**
Added notes for _Advances in UI Data Sources_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -911,3 +912,54 @@ https://developer.apple.com/wwdc19/615
...
+- **Identifiers**
+  - Must be unique (UUID)
+  - Conforms to Hashable
+  - Data model or identifier
+- **Performance**
+  - Fast
+  - Safe to call `apply()` from a background queue
+    - Although always call exclusively from the main queue or a background queue and stay consistent (if you choose to go background, always go background)
+- **apply()**
+  - Available for iOS, tvOS and macOS
```
Comment (Blackjacx) [inline README.md:963]: Not for watchOS?

**Review by Blackjacx — APPROVED** [review]

**Comment by Blackjacx** [comment]
Nice thanks for your contribution. You can add your GitHub name to the contributors section if you want :-)

**Comment by Blackjacx** [comment]
> Nice thanks for your contribution. You can add your GitHub name to the contributors section if you want :-)

Oh I now merged it - I'll add you :)

File: README.md (hunk 1 above)
Comment (soucolline) [reply README.md:963]: Unfortunately no, watchOS does not use `UITableView` or `UICollectionView` as in iOS but has a specific `WKInterfaceTable` (no collection view apparently) which is lacking most of the features we can have in iOS

**Pull request merged**